Apex Language/APEX_2/AccountUtil.cls

```diff
@@ -0,0 +1,19 @@
+public class AccountUtil {
+    public static void storeContactIds(Id contactId){
+    	Contact contact = [SELECT accountid FROM Contact WHERE id =: contactId];
```

Comment:
`contactRecord` because contact / Contact it may become ambiguous

Reply:
done it.